src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Not quite what I'm accurately stating (not arguing). The CA bundle is handled via pkg ( ca_root_nss ) so it does not need to be in config.xml. The user has their CAs in config.xml and we need to be sure the CAs written in /etc/ssl/ are the CAs from config.xml. They don't come from other sources.

We need to be absolutely certain that the files match config.xml. Either by overwriting or completely and accurately verifying their contents.

If you want to handle your own system such that you could have an inconsistent state or corrupt files in /etc/ssl/ go ahead, but for code we include, it has to be ensured to be consistent at boot time.